recipe/meta.yaml

```diff
@@ -32,7 +37,7 @@ requirements:
     - backcall
     - colorama  # [win]
     - decorator
-    - jedi >=0.16,<1.0
```

Comment:
Did the upstream project fix their compatibility issues with Jedi 1.0? If not, we should probably keep the upper bound pinned.

Reply:
The upstream setup.py file has `jedi >=0.16` https://github.com/ipython/ipython/blame/7.31.0/setup.py#L198
I found only this backport PR https://github.com/ipython/ipython/commit/5e979ec2119487e0d6ac33292f07ab62826645a5 and the PR https://github.com/ipython/ipython/pull/12207

On conda-forge this pinning was removed starting with v7.30.0 too https://github.com/conda-forge/ipython-feedstock/commit/b64cd1576d72af5be930f2e05202e38e9936fe5f